src/main/java/uk/gov/hmcts/reform/professionalapi/controller/response/ProfessionalUsersEntityResponse.java

```diff
@@ -21,4 +25,12 @@
     public void setUserProfiles(List<ProfessionalUsersResponse> userProfiles) {
         this.userProfiles = userProfiles;
     }
+
+    public void setOrganisationIdentifier(String organisationIdentifier) {
+        this.organisationIdentifier = organisationIdentifier;
+    }
+
+    public String getOrganisationIdentifier() {

```

Comment:
getter is required here?

Reply:
yes required for object to Json parsing.

Reply:
ok